examples/ex_1_from_qasm.py

```diff
@@ -57,4 +69,6 @@ def main(file_name):
 
 if __name__ == "__main__":
     current_directory = os.path.dirname(__file__)
+    # main(current_directory + "/data/single_qubit_clifford_circuit.qasm")
```

Comment:
Let's remove this since the file doesn't exist.
```suggestion
```

Reply:
Fixed!